Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/organizer/details/QrCodeDetailFragment.kt

```diff
@@ -152,6 +159,20 @@ class QrCodeDetailFragment : Fragment(R.layout.trace_location_organizer_qr_code_
         behavior.overlayTop = ((width) / 2) - 24
     }
 
+    private fun openCreateEventFragment(traceLocation: TraceLocation) {
+        val category = traceLocationCategories.find { it.type == traceLocation.type }
+        if (category == null) {
+            Timber.e("Category not found, traceLocation = $traceLocation")
+        } else {
+            findNavController().navigate(
+                QrCodeDetailFragmentDirections.actionQrCodeDetailFragmentToTraceLocationCreateFragment(
+                    category,
+                    traceLocation
+                )
+            )
+        }
+    }
+
```

Comment:
could you please pass the category directly from the view model in the event?  you don't have to do this in the fragment `NavigateToDuplicateFragment(val traceLocation: TraceLocationCategory)`

Reply:
ok, updated